libs/web-components/src/components/calendar/Calendar.svelte

```diff
@@ -297,7 +298,7 @@
   tabindex="-1"
 >
   <goa-block mb="m">
-    <goa-form-item label="Month" mt="0">
+    <goa-form-item label="Month" mt="0" {version}>
```

Comment:
FormItem labelsize should match compact dropdowns in V2.

```suggestion
    <goa-form-item label="Month" mt="0" {version} labelsize={version === "2" ? "compact" : "regular"}>
```